README.md

```diff
@@ -57,89 +57,113 @@ export default {
 
 ## Options
 
-You can pass [eslint options](http://eslint.org/docs/developer-guide/nodejs-api#cliengine).
+You can pass [eslint options](https://eslint.org/docs/developer-guide/nodejs-api#%E2%97%86-new-eslint-options).
 
-**Note**: That the config option you provide will be passed to the `CLIEngine`. This is a different set of options than what you'd specify in `package.json` or `.eslintrc`. See the [eslint docs](http://eslint.org/docs/developer-guide/nodejs-api#cliengine) for more detail.
+Note that the config option you provide will be passed to the `ESLint` class.
+This is a different set of options than what you'd specify in `package.json` or `.eslintrc`.
+See the [eslint docs](https://eslint.org/docs/developer-guide/nodejs-api#%E2%97%86-new-eslint-options) for more details.
 
-### `cache`
+**Warning**: In eslint-webpack-plugin version 1 the options were passed to the now deprecated [CLIEngine](https://eslint.org/docs/developer-guide/nodejs-api#cliengine).
 
-- Type: `Boolean|String`
-- Default: `false`
+### `cache``
+
+- Type: `Boolean`
+- Default: `true`
 
-This option will enable caching of the linting results into a file. This is particularly useful in reducing linting time when doing a full build.
+**Note**: The cache is enabled by default to decrease execution time.
 
-This can either be a `boolean` value or the cache directory path(ex: `'./.eslint-loader-cache'`).
+### `context`
 
-If `cache: true` is used, the cache is written to the `./node_modules/.cache/eslint-loader` directory. This is the recommended usage.
+- Type: `String`
+- Default: `compiler.context`
+
+A string indicating the root of your files.
 
 ### `eslintPath`
 
 - Type: `String`
 - Default: `eslint`
 
-Path to `eslint` instance that will be used for linting. If the `eslintPath` is a folder like a official eslint, or specify a `formatter` option. Now you dont have to install `eslint`.
+Path to `eslint` instance that will be used for linting. If the `eslintPath` is a folder like a official eslint, or specify a `formatter` option. now you dont have to install `eslint`.
```

Comment:
This paragraph is a bit broken grammatically. Can you rephrase it?

Reply:
Fixed. Thanks for review